Directory.Packages.props

```diff
@@ -0,0 +1,44 @@
+<Project>
+  <!-- See https://learn.microsoft.com/nuget/consume-packages/Central-Package-Management -->
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageVersion Include="CommandLineParser" Version="1.9.71" />
+    <PackageVersion Include="HtmlAgilityPack" Version="1.11.46" />
+    <PackageVersion Include="ICSharpCode.Decompiler" Version="8.0.0.7246-preview3" />
+    <PackageVersion Include="iTextSharp" Version="5.5.13.3" />
+    <PackageVersion Include="Jint" Version="3.0.0-beta-2048" />
+    <PackageVersion Include="JsonSchema.Net" Version="4.0.0" />
+    <PackageVersion Include="Markdig" Version="0.31.0" />
+    <PackageVersion Include="Microsoft.Build" Version="17.5.0" />
+    <PackageVersion Include="Microsoft.Build.Locator" Version="1.5.5" />
+    <PackageVersion Include="Microsoft.CodeAnalysis" Version="4.5.0" />
+    <PackageVersion Include="Microsoft.CodeAnalysis.Common" Version="4.5.0" />
+    <PackageVersion Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="4.5.0" />
+    <PackageVersion Include="Microsoft.CodeAnalysis.CSharp" Version="4.5.0" />
+    <PackageVersion Include="Microsoft.CodeAnalysis.VisualBasic.Workspaces" Version="4.5.0" />
+    <PackageVersion Include="Microsoft.CodeAnalysis.VisualBasic" Version="4.5.0" />
+    <PackageVersion Include="Microsoft.CodeAnalysis.Workspaces.Common" Version="4.5.0" />
+    <PackageVersion Include="Microsoft.CodeAnalysis.Workspaces.MSBuild" Version="4.5.0" />
+    <PackageVersion Include="Microsoft.SourceLink.GitHub" Version="1.1.1" />
```

Comment:
Maybe this can be specified as a `GlobalPackageReference`

Reply:
Aha, I don't have too much preference. Not using central package version management at all is also good to me because @dependabot takes care of all the jobs and I almost never update these packages myself 😁

Reply:
I actually never seen that property. Though, according to the docs it's appropriate for references that apply to all projects. Not sure, if any of these truly are.

Reply:
Looks good from my side, I'm using this in [my project](https://github.com/WeihanLi/dotnet-exec/blob/0.13.0/Directory.Packages.props#L26) and the NuGet package has source link info also https://www.nuget.info/packages/dotnet-execute/0.13.0.

![image](https://user-images.githubusercontent.com/7604648/226527790-9e301cbd-e72c-4cdd-93c8-c800902ab782.png)